polyfill.js

```diff
@@ -0,0 +1,9 @@
+import DurationFormat from './index.js';
+
+if (typeof Intl === 'undefined') {
+  globalThis.Intl = {};
+}
+
+if (typeof Intl.DurationFormat === 'undefined') {
+  Intl.DurationFormat = DurationFormat;
```

Comment:
I'm not sure about this line. The official proposal for DurationFormat diverged significantly from my implementation. It might be confusing to provide a polyfill for something that works so differently.
What do you think?

I haven't checked the status recently but if the official proposal made some progress maybe we should consider converging towards it or even deprecating this library in favor of the official polyfill.
Since we have a breaking change due to node 10 we can also change the public API

Reply:
Yeah, I have since seen that there is more to polyfill as per #21, and even then a polyfill might not be in order until it is standardized and the fallback behavior can be expected to be consistent. So maybe we can just hold off on this PR and try to support #21 first (since I don't think there is any official polyfill to do it for us anyways). Or I could remove that portion and any additional support could be built on top. Let me know what you may prefer. Thanks!